### PR TITLE
fix [aws-ec2]: update the AwsEc2 nodesource example

### PR DESCRIPTION
**Pull request by tao-xinxiu**

### CODE REVIEW

File: NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+        "value": "[please put your AWS secret access key]",
+        "meta": {
+          "type": "NONE",
+          "description": "Your AWS secret access key (e.g., wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY)",
+          "dynamic": false
+        }
+      },
+      {
+        "name": "numberOfInstances",
+        "value": "10",
```
Comment (MaelAudren) [inline NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json]: Isn't that a big number ?

File: NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+          "description": "(optional) Your AWS private key file corresponding to 'vmKeyPairName' for accessing VM",
+          "dynamic": false
+        }
+      },
+      {
+        "name": "ram",
+        "value": "2048",
+        "meta": {
+          "type": "NONE",
+          "description": "(optional) The minimum RAM required (in Mega Bytes) for each VM",
```
Comment (MaelAudren) [inline NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json]: Why not specifying the default value unlike upper parameter ? Same comment for next parameters.

File: NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+          "description": "Desired number of tasks per node.",
+          "dynamic": false
+        }
+      },
+      {
+        "name": "initDelay",
+        "value": "120000",
+        "meta": {
+          "type": "NONE",
+          "description": "Delay to initialize the infrastructure (eg. in a scaleSet this must cover the creation of Azure's resources).",
```
Comment (MaelAudren) [inline NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json]: Add the (ms) to specify the unit used.

File: NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json (hunk 1 above)
Comment (tao-xinxiu) [reply NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json]: It's the maximum number of instances, when it's smaller than what's required by the `maxNodes`, the smaller one (in the example, maxNodes requires 5 instances at most, and it's updatable) is applied.

File: NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json (hunk 2 above)
Comment (tao-xinxiu) [reply NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json]: Agreed, but I should add it in the nodesource-addons project (the origin of the parameters description).
As the PR of nodesource-addons is already merged, I'm not sure we do it now or wait until next release?

File: NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json (hunk 3 above)
Comment (tao-xinxiu) [reply NodeSources/resources/catalog/AwsEC2_with_dynamic_policy.json]: This description should be updated in the `DynamicPolicy` (project enterprise-nodesource-addons). I think it's not a good idea to use different description between the policy/infrastructure and the example.

**Comment by tao-xinxiu** [comment]
> Please, open issues in the related projects regarding the improvement that we highlighted.

Done. 
https://github.com/ow2-proactive/nodesource-addons/issues/72
https://bitbucket.org/activeeon/enterprise-nodesource-addons/issues/1/improve-parameters-description

**Pull request merged**